tests/shell/shell_recorder/script.dat

```diff
@@ -0,0 +1,13 @@
+Storage: dump
+Mountpoint: system/testmount/test
+File: /tmp/test.dump
+MountArgs:
```

Comment:
If I understand it correctly "Storage:" is the specification in which format the recorder does import/export. What do we need MountArgs for?